### PR TITLE
Added support for New Relic

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
This adds support for New Relic.

### How to review 
When the NEW_RELIC_LICENSE_KEY is supplied then data should be shown in New Relic

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -256,3 +256,5 @@ WTForms==2.1 \
     --hash=sha256:ffdf10bd1fa565b8233380cb77a304cd36fd55c73023e91d4b803c96bc11d46f
 Flask_WTF==0.14.2 \
     --hash=sha256:d9a9e366b32dcbb98ef17228e76be15702cd2600675668bca23f63a7947fd5ac
+newrelic==2.78.0.57 \
```
Comment (collisdigital) [inline requirements.txt:259]: should this only be installed for test? i.e. move to the other reqs file?

Comment (jonnyshaw89) [reply requirements.txt:259]: I don't think it works like that. the other file is only used for unit / integration tests?
This in theory isn't a test dependency as we want it in the code all the time.

**Review by ajmaddaford — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (collisdigital) [reply requirements.txt:259]: I guess there's no simple way of dynamically installing dependencies based on your build options...

Comment (jonnyshaw89) [reply requirements.txt:259]: I guess then the argument would be that the tested build isn't the build you run in production?

**Review by LJBabbage — APPROVED** [review]

**Pull request merged**